### PR TITLE
docs: edits related to loadList

### PR DESCRIPTION
**Pull request by rasahner**
In response to issue #2069, I looked at the build.md page.  Having gone straight there, I was confused about the instructions for what to do with the mysql .tar.  I don't think this is what was causing someone
hours of trouble, but it led me to try to make some improvements.

### CODE REVIEW

File: docs/content/development/build.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ mvn clean install
 This will compile the project and create the Druid binary distribution tar under
 `distribution/target/druid-VERSION-bin.tar.gz`.
 
-This will also create a tarball that contains `mysql-metadata-storage` extension under
- `distribution/target/mysql-metadata-storage-bin.tar.gz`. If you want Druid to load `mysql-metadata-storage`, you can first untar `druid-VERSION-bin.tar.gz`, then go to ```druid-<version>/extensions```, untar `mysql-metadata-storage-bin.tar.gz` there. Now just specifiy `mysql-metadata-storage` in `druid.extensions.loadList` so that Druid will pick it up. See [Including Extensions](../operations/including-extensions.html) for more infomation.
+This will also create `distribution/target/mysql-metadata-storage-bin.tar.gz`,
```
Comment (rasahner) [inline docs/content/development/build.md]: I thought the removed information was out of place here, and would better be placed in the metadata storage page.

**Comment by nishantmonu51** [comment]
:+1:

File: docs/content/dependencies/metadata-storage.md [hunk 2]
```diff
@@ -12,17 +12,22 @@ a number of tables used for various purposes described below.
...
-
+* MySQL
+* PostgreSQL
+
+To choose the metadata storage type, set `druid.metadata.storage.type` to `derby`,
+`mysql` or `postgres`. Set other `druid.metadata.storage` configuration
+keywords as shown below to give Druid information about how to connect to
+the database.
+
+As discussed in [Including Extensions](../operations/including-extensions.html),
```
Comment (rasahner) [inline docs/content/dependencies/metadata-storage.md]: The previous instructions seemed to be saying you'd need to set the druid.extensions configuration keyword even if you were picking up the extension files from the classpath.

File: docs/content/dependencies/metadata-storage.md [hunk 3]
```diff
@@ -57,20 +62,20 @@ metadata storage you plan to use. See below.
...
   druid.extensions.loadList=["mysql-metadata-storage"]
   druid.metadata.storage.type=mysql
-  druid.metadata.storage.connector.connectURI=jdbc:mysql://<host>/druid_test
+  druid.metadata.storage.connector.connectURI=jdbc:mysql://<host>/druid
   druid.metadata.storage.connector.user=druid
   druid.metadata.storage.connector.password=diurd
   ```
 
-  Note: metadata storage extension is not packaged within the main Druid tarball, it is
+  Note: the metadata storage extension is not packaged within the main Druid tarball; it is
```
Comment (rasahner) [inline docs/content/dependencies/metadata-storage.md]: I did not find a metadata storage tarball at the link.  That link is used for this purpose on the first-look tutorials page, and maybe elsewhere.

File: docs/content/dependencies/metadata-storage.md (hunk 2 above)
Comment (fjy) [inline docs/content/dependencies/metadata-storage.md]: please mention derby is not suitable for production

Comment (rasahner) [inline docs/content/dependencies/metadata-storage.md]: done

**Comment by fjy** [comment]
:+1:

**Pull request merged**